css/partials/_breadcrumbs.scss

```diff
@@ -17,12 +17,12 @@
 //
 // Begin old styles. Clean this stuff up, when time.
 //
-#breadcrumb {
+.breadcrumbs {
```

Comment:
standardize breadcrumbs and make visual left alignment consistent (breadcrumb, page header, and content should all be visually aligned with this change).